scaleway/scaleway/vpc/v2/tests/cassettes/test_vpc_get.cassette.yaml

```diff
@@ -0,0 +1,41 @@
+interactions:
+- request:
+    body: '{}'
+    headers:
+      Content-Length:
+      - '2'
+      user-agent:
+      - scaleway-sdk-python/2.0.0
+    method: GET
+    uri: https://api.scaleway.com/vpc/v2/regions/fr-par/vpcs/7dfbbfc0-3f5a-4cee-b40b-4ef73ddda974
+  response:
+    body:
+      string: '{"id": "7dfbbfc0-3f5a-4cee-b40b-4ef73ddda974", "name": "vpc-test-sdk-python-practical-beaver",
+        "tags": [], "organization_id": "11111111-1111-1111-1111-111111111111", "created_at":
+        "2025-08-29T07:51:47.085349Z", "updated_at": "2025-08-29T07:51:47.085349Z",
+        "project_id": "11111111-1111-1111-1111-111111111111", "is_default": false,
+        "private_network_count": 0, "routing_enabled": true, "custom_routes_propagation_enabled":
+        true, "region": "fr-par"}'
+    headers:
+      content-length:
+      - '423'
+      content-security-policy:
+      - default-src 'none'; frame-ancestors 'none'
+      content-type:
+      - application/json
+      date:
+      - Fri, 29 Aug 2025 07:51:57 GMT
+      server:
+      - Scaleway API Gateway (fr-par-2;edge01)
+      strict-transport-security:
```

Comment:
Would it be possible to clean up those headers? It would be helpful to have a list of headers that should be removed when persisted on cassettes